File: 485-max-consecutive-ones/max-consecutive-ones.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findMaxConsecutiveOnes = function(nums) {
    let currentCount= 0;
    let maxCount =0

    for(i=0; i<nums.length; i++){
        if (nums[i]==1){
            currentCount++;
        }
        else{
            maxCount= Math.max(maxCount,currentCount);
            currentCount=0;
        }
        
    }
    return Math.max(maxCount,currentCount);
    
};

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
      int CurrentCount = 0;
      int MaxCount = 0;

    for(int i =0; i<nums.length; i++){
        if(nums[i]==1){
            CurrentCount++;

            MaxCount = Math.max(CurrentCount,MaxCount);
        }
        else{
            CurrentCount =0 ;
        }
    }
    return MaxCount;
    }
}